src/mmda/predictors/__init__.py

```diff
@@ -1,10 +1,12 @@
 # flake8: noqa
 from necessary import necessary
 
-from mmda.predictors.heuristic_predictors.whitespace_predictor import WhitespacePredictor
-from mmda.predictors.heuristic_predictors.dictionary_word_predictor import DictionaryWordPredictor
+with necessary(["tokenizers"], soft=True) as TOKENIZERS_AVAILABLE:
```

Comment:
in what case would we run this without the tokenizers?

Reply:
The Bib Detector model doesn't need the 'tokenizers' library, the reason I added this in is because when I ran tt verify at first, it errored because DictionaryWordPredictor required 'tokenizers' - but the Bib Detector model doesn't need DictionaryWordPredictor in the first place!